projects/Mallard/android/app/build.gradle

```diff
@@ -78,7 +78,7 @@ import com.android.build.OutputFile
 
 project.ext.react = [
     entryFile: "index.js",
-    enableHermes: false,  // clean and rebuild if changing
+    enableHermes: true,  // clean and rebuild if changing
```

Comment:
What is this/does this do?

Reply:
https://facebook.github.io/react-native/docs/hermes/

Reply:
im testing it out – do not merge